lib/chef-dk/policyfile_services/export_repo.rb

```diff
@@ -85,13 +109,35 @@ def export
 
       private
 
-      def create_repo_structure
-        FileUtils.rm_rf(cookbooks_dir)
-        FileUtils.rm_rf(policyfiles_data_bag_dir)
+      def with_staging_dir
+        p = Process.pid
+        t = Time.new.utc.strftime("%Y%m%d%H%M%S")
+        Dir.mktmpdir("chefdk-export-#{p}-#{t}") do |d|
```

Comment:
it seems fine. It should put it in applocal things. Is there something I can test out for you? Dir.mktmpdir ran without issue